scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixArguments.java

```diff
@@ -169,6 +164,7 @@ ScalafixArguments withToolClasspath(
      *                this must match the binary version available in the classloader of
      *                this instance, as requested/provided in the static factory methods
      *                of {@link Scalafix}.
+     * @throws ScalafixException In case of an error parsing the scalaVersion.
```

Comment:
the javadoc could be updated in the same way the CLI doc was updated

Reply:
ok

Reply:
the cli sentence is finally not that clear for me!
```
    @Description(
      "The Scala compiler version that the provided files are targeting, " +
        "or the actual version that was used to compile them when a classpath is provided."
    )
```
what we want is 
- the full scalaverion that was used to compile the code source
- in the absence of the fullVersion, we require either the scala binary version of the code source, or the major scala Version

Reply:
we need at least the major, to know with which dialect to parse the code source, but some rules may require more..

Reply:
ok!